tests/Unit/Rule/RequestAttributeValidationRule/Fixture/UseDefinedRequestAttribute.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types = 1);
+
+namespace SaschaEgerer\PhpstanTypo3\Tests\Unit\Rule\RequestAttributeValidationRule\Fixture;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+final class UseDefinedRequestAttribute
+{
+
+	public function someMethod(): void
+	{
+		$this->getServerRequest()->getAttribute('backend.user');
+	}
+
+	public function getServerRequest(): ServerRequestInterface
+	{
+		return $GLOBALS['TYPO3_REQUEST'];
```

Comment:
Are we sure we can use this one in a unit test? Should we use a mock instead?

Reply:
Yes, it is statically analysed. We are just pretending that an object of Type ServerRequestInterface is given.